tests/west/commands/test_commands.py

```diff
@@ -282,7 +298,6 @@ def test_manifest_freeze(west_update_tmpdir):
     for eline_re, aline in zip(expected_res, actual):
         assert re.match(eline_re, aline) is not None, (aline, eline_re)
 
-
```

Comment:
Why is this line removed ?

Reply:
Too much whitespace ;)

Reply:
I thought you were using a black background on your screen.

Reply:
Could you please put the blank line back ?
I believe we are following pep8 for python code:
https://www.python.org/dev/peps/pep-0008/

> Blank Lines
>
> Surround top-level function and class definitions with two blank lines.